scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -157,15 +157,15 @@ function SdAddEmbedController (embedService, $element, $timeout, $q, _, EMBED_PR
     // toggle when the `extended` directive attribute changes
     $scope.$watch(function() {
         return vm.extended;
-    }, function(extended) {
+    }, function(extended, wasExtended) {
```

Comment:
@gbbr fyi it's just current and previous value returned from that watched function, so value of `vm.extended`. you can find more info about `$watch` in [angular docs](https://docs.angularjs.org/api/ng/type/$rootScope.Scope)